cdap-flume/src/main/java/co/cask/cdap/flume/StreamSink.java

```
@@ -128,7 +185,31 @@ private void createStreamClient() throws IOException {
 
       builder.writerPoolSize(writerPoolSize);
       builder.version(version);
+      AuthenticationClient authClient = null;
+      InputStream inStream = null;
+      try {
+        authClient = (AuthenticationClient) Class.forName(authClientClassPath).getConstructor().newInstance();
+        authClient.setConnectionInfo(host, port, sslEnabled);
+        Properties properties = new Properties();
+        inStream = new FileInputStream(authClientPropertiesPath);
+        properties.load(inStream);
+        authClient.configure(properties);
+        builder.authClient(authClient);
+      } catch (ReflectiveOperationException e) {
+        LOG.error("Can not resolve class {}: {}", new Object[]{authClientClassPath, host, port, e.getMessage(), e});
```

So you just log and ignore the error? That is, you will proceed as if no authentication was configured.